create_elasticsearch_secrets.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+create_namespace() {
+    local context=$1
+    local namespace=$2
+    if kubectl --context "$context" get namespace "$namespace" &> /dev/null; then
+        echo "Namespace $namespace already exists."
+    else
+        # Create the namespace
+        kubectl --context "$context" create namespace "$namespace"
+    fi
+}
+
+create_secret() {
+    local context=$1
+    local namespace=$2
+    local secret_name=$3
+    local access_key=$4
+    local secret_access_key=$5
+    kubectl --context "$context" -n "$namespace" create secret generic "$secret_name" \
+        --from-literal=S3_ACCESS_KEY="$access_key" \
+        --from-literal=S3_SECRET_KEY="$secret_access_key"
+}
+
+if [ -z "$ACCESS_KEY" ]; then
```

Comment:
suggestion: use `AWS_ACCESS_KEY` and `AWS_SECRET_ACCESS_KEY` to explicitly show this is only for AWS installations.

Reply:
changed and added a suffix `_ES` as otherwise it will potentially clash with the AWS CLI config and cause more harm than it does good.